### PR TITLE
UIDATIMP-429: Fix Accessibility problems for /data-import page

### PR DESCRIPTION
**Pull request by IhorKoba**
## Purpose
To fix all Accessibility problems for /data-import page

## Approach
- Change tag inside list from `<div>` to `<li>` in `Job` component
- Update styles

## Ticket
[UIDATIMP-429](https://issues.folio.org/browse/UIDATIMP-429)

### CODE REVIEW

File: src/components/Jobs/components/Job/Job.css [hunk 1]
```diff
@@ -20,6 +20,15 @@
   }
 }
 
+[class*="listStyleDefault"] {
```
Comment (mariia-aloshyna) [inline src/components/Jobs/components/Job/Job.css:23]: Will it work for `.listStyleDefault`?

Comment (IhorKoba) [reply src/components/Jobs/components/Job/Job.css:23]: This styles apply only for `.job` inside `. listStyleDefault ` and it's only this case.

Comment (mariia-aloshyna) [reply src/components/Jobs/components/Job/Job.css:23]: I mean will it work if `[class*="listStyleDefault"]` replace with `.listStyleDefault`?

Comment (IhorKoba) [reply src/components/Jobs/components/Job/Job.css:23]: no, because of hash added to styles for parent component

Comment (mariia-aloshyna) [reply src/components/Jobs/components/Job/Job.css:23]: Got it, thanks

**Pull request merged**